chrono-tz-build/src/lib.rs

```diff
@@ -430,6 +433,26 @@ mod filter {
     }
 }
 
+fn detect_iana_db_version() -> String {
```

Comment:
How do you feel about doing some parsing of the `NEWS` file instead? Read lines, stop on the first line that starts with `Release` and take the next word as the version. I don't love this git repo mutation stuff in library code.

Reply:
I agree that is safer. We can always fix the parsing if the format changes.